x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/feature_importance/decision_path_classification.tsx

```diff
@@ -42,12 +43,20 @@ export const ClassificationDecisionPath: FC<ClassificationDecisionPathProps> = (
   const [currentClass, setCurrentClass] = useState<string>(
     getStringBasedClassName(topClasses[0].class_name)
   );
+  const selectedClass = topClasses.find(
```

Comment:
The description of the fix refers to `True` and `False` base classes, but is this change also the cause of the change in behavior here with the `bank_classification_1` MLQA bootstrap job where before I was seeing values greater than `1` for `yes` and `no` classes:

Before:

<img width="1042" alt="image" src="https://user-images.githubusercontent.com/7405507/218087391-9d02aceb-efbf-430e-9a2b-66a366d65d51.png">


With this fix:

<img width="1116" alt="image" src="https://user-images.githubusercontent.com/7405507/218087632-8fcf5fbc-3275-48fc-a53e-46015bb218bb.png">

Reply:
AFAICS, the fix makes total sense here: for the second row, the model predicts the class "no" with the probability of 0.986. This means that the prediction probability of "yes" is 0.014. This is precisely what the screenshot above shows now 🚀  

The screenshot before this fix shows that the decision graph briefly exceeds 1.0 in for prediction probability, which is nonsense.